src/components/ActionButtons/ButtonBase.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { func, oneOfType, object, node } from 'prop-types';
+import { useSpring, interpolate } from 'react-spring';
+
+import Button from '../Button';
+
+function ButtonBase({ Icon, onClick = () => {}, children, ...props }) {
```

Comment:
I would change **I**con to **i**con and afterwards make an alias `icon: Icon`. Capital name of prop looks really weird, even if we consider that it expects component.